src/pages/events/configure-commerce.md

```diff
@@ -142,14 +142,150 @@ Commerce provides two sources for events: observers and plugins. You must specif
 
 1. Optionally create a new OAuth or JWT credential. Then click **Next**.
 
-1. Update the **Event registration name** and **Event registration description** fields. In the **How to receive events** section, under **Option 2**, select the runtime action you created in [Set up App Builder and define a runtime action](./project-setup.md#set-up-app-builder-and-define-a-runtime-action).
-
-   ![Select a runtime action](../_images/events/select-runtime-action.png)
+1. Update the **Event registration name** and **Event registration description** fields.
 
 1. Select **Save configured events**.
 
 You are now set up to develop your App Builder extension.
 
+## (Optionally) Set up App Builder project with event registration and runtime actions
+
+The first step to setting up your App Builder template is to set up your environment and create a runtime action. For details about this process, see [Setting up Your Environment](https://developer.adobe.com/runtime/docs/guides/getting-started/setup/) in the _Adobe IO Runtime Guide_ and [Create a templated project](https://developer.adobe.com/developer-console/docs/guides/projects/projects-template/) in the _Developer Console Guide_.
```

Comment:
Because it's not a step that has to be taken, I'm wondering if the first sentence should be reworded to start with something like "If you want to add runtime actions..."

Reply:
Makes sense, I changed it, but maybe it can be paraphrased in a better way.